### PR TITLE
DOCS: Adding a portlist for biergit

### PR DESCRIPTION
**Pull request by ThomasWilczek**

### CODE REVIEW

**Comment by ThomasWilczek** [comment]
Not all ports are listed yet.
Please have a look @m-Schlitzer

File: BIERGIT_PORTS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Biergit port overwiev
```
Comment (m-Schlitzer) [inline BIERGIT_PORTS.md]: overwiev -> overview

**Comment by dreisen** [comment]
Don't forget to squash before merging

**Comment by m-Schlitzer** [comment]
Otherwise, these are all ports that are in use right now. Should we switch gitlab to 8080 when nginx is running on 80?

**Comment by m-Schlitzer** [comment]
Also, remember our guide to git issues next time please.

**Pull request closed without merging**